### ISSUE TITLE
Twenty Twenty-Two and Twenty-Three: Details block icon is described with screen reader 

### ISSUE THREAD

**Issue by likethegoddess** [Automattic/themes#7302]
### Quick summary

The Details block icon is described as "Filled right-pointing small triangle" when using NVDA on PC with Firefox. The icon should be ignored by screen readers. 

### Steps to reproduce

1) Open NVDA on PC 
2) Go to page with Detail block added
3) Have app read page or hover over icon

### What you expected to happen

Only the link text, "button," and "expanded" or "collapsed" should be read by a screen reader. 

### What actually happened

"Filled right-pointing small triangle" is read followed by remaining information.  

### Browser

Mozilla Firefox

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

None
